### PR TITLE
Add documentation and support links

### LINKED ISSUE #2770: Add links to Documentation in the console
#### Summary
Add links to our Documentation.

#### Why do we need this?
People don't know/forget the right link to our docs. 

#### What is missing? What do you want to see?
Add an extra line at the landing page.
Screenshot 2020-06-19 at 09 26 28

And in the footer:
Screenshot 2020-06-19 at 09 23 39

TTS link: https://thethingsstack.io/
TTES link: https://enterprise.thethingsstack.io/
Get support link: https://www.thethingsindustries.com/support

#### Environment
v3.8.3

#### How do you propose to implement this?
Add the link

#### How do you propose to test this?
Test on local setup.

### PR DESCRIPTION
**Pull request by asmulko**
#### Summary

Closes #2770 

#### Changes

- Added documentation and support links in welcome page
- Added documentation link to footer

Before:
before_1
before_2

After:
after_1
after_2

If no support link provided in configuration:
without_support_link

#### Testing

Manually tested

#### Checklist
- [x] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Compatibility: The changes are backwards compatible with existing API, storage, configuration and CLI, according to the compatibility commitments in `README.md` for the chosen target branch.
- [ ] Documentation: Relevant documentation is added or updated.
- [ ] Changelog: Significant features, behavior changes, deprecations and fixes are added to `CHANGELOG.md`.
- [x] Commits: Commit messages follow guidelines in `CONTRIBUTING.md`, there are no fixup commits left.

### CODE REVIEW

File: pkg/webui/components/footer/index.js [hunk 1]
```diff
@@ -50,9 +50,16 @@ const Footer = function({ className, links, supportLink, isOnline }) {
         ))}
         <OfflineStatus isOnline={isOnline} showOfflineOnly showWarnings />
         <span className={style.version}>v{process.env.VERSION}</span>
+        <Link.Anchor
+          className={style.documentation}
+          secondary
+          href="https://www.thethingsindustries.com/docs"
```
Comment (bafonins) [inline pkg/webui/components/footer/index.js]: This should be taken from the configuration (`APP_CONFIG.documentation_base_url`).
See https://github.com/TheThingsNetwork/lorawan-stack/blob/v3.10/pkg/webui/lib/env.js#L30

File: pkg/webui/console/views/overview/index.js [hunk 2]
```diff
@@ -247,6 +248,23 @@ export default class Overview extends React.Component {
                   component="h2"
                 />
               )}
+              <Message
+                className={style.getStarted}
+                content={m.needHelp}
+                values={{
+                  documentationLink: (
+                    <Link.Anchor secondary href="https://www.thethingsindustries.com/docs">
```
Comment (bafonins) [inline pkg/webui/console/views/overview/index.js]: Use url from the configuration

**Review by bafonins — CHANGES_REQUESTED** [review]
See my comments.

Please submit additional PR to ttes with similar fixes/changes.

File: pkg/webui/console/views/overview/index.js [hunk 3]
```diff
@@ -247,6 +248,23 @@ export default class Overview extends React.Component {
                   component="h2"
                 />
               )}
+              <Message
+                className={style.getStarted}
+                content={m.needHelp}
+                values={{
+                  documentationLink: (
+                    <Link.Anchor secondary href={documentationBaseUrl}>
```
Comment (bafonins) [inline pkg/webui/console/views/overview/index.js]:
```suggestion
                    <Link.Anchor secondary href={documentationBaseUrl} target="_blank">
```

We want to open a new tab for the user

File: pkg/webui/console/views/overview/index.js [hunk 4]
```diff
@@ -247,6 +248,23 @@ export default class Overview extends React.Component {
...
+                      <Message content={sharedMessages.documentation} />
+                    </Link.Anchor>
+                  ),
+                  supportLink: (
+                    <Link.Anchor secondary href={supportLink}>
```
Comment (bafonins) [inline pkg/webui/console/views/overview/index.js]:
```suggestion
                    <Link.Anchor secondary href={supportLink} target="_blank">
```

@laurensslats Do we really need the support link in ttn?

Comment (bafonins) [reply pkg/webui/console/views/overview/index.js]: Please also check if `supportLink` exists similar to how its done is footer. Currently, with default configuration `supportLink` is empty.

**Review by bafonins — CHANGES_REQUESTED** [review]

File: pkg/webui/console/views/overview/index.js [hunk 5]
```diff
@@ -58,6 +58,7 @@ const m = defineMessages({
   createGateway: 'Register a gateway',
   gotoApplications: 'Go to applications',
   gotoGateways: 'Go to gateways',
+  needHelp: 'Need help? Have a look at our {documentationLink} or {supportLink}',
```
Comment (kschiffer) [inline pkg/webui/console/views/overview/index.js]:
```suggestion
  needHelp: 'Need help? Have a look at our {documentationLink} or {supportLink}.',
```

File: pkg/webui/console/views/overview/index.js [hunk 6]
```diff
@@ -58,6 +58,7 @@ const m = defineMessages({
...
   welcome: 'Welcome to the Console!',
   welcomeBack: 'Welcome back, {userName}! 👋',
   getStarted: 'Get started right away by creating an application or registering a gateway',
```
Comment (kschiffer) [inline pkg/webui/console/views/overview/index.js]:
```suggestion
  getStarted: 'Get started right away by creating an application or registering a gateway.',
```
Please change L64 accordingly as well.

File: pkg/webui/console/views/overview/index.js (hunk 3 above)
Comment (kschiffer) [inline pkg/webui/console/views/overview/index.js]: This should actually be a `<Link.DocLink>`
```suggestion
                    <Link.DocLink secondary path="/" external>
```
To make this work you need to allow custom titles for DocLinks.

File: pkg/webui/console/views/overview/index.js (hunk 4 above)
Comment (kschiffer) [inline pkg/webui/console/views/overview/index.js]: We have an `external` prop for that:
```suggestion
                    <Link.Anchor secondary href={supportLink} external>
```

**Review by kschiffer — CHANGES_REQUESTED** [review]

File: pkg/webui/console/views/overview/index.js (hunk 4 above)
Comment (asmulko) [reply pkg/webui/console/views/overview/index.js]: I implemented it the following way: If no `supportLink` is going to be added in the configuration, then `need help `text will change (see last screenshot)

Comment (asmulko) [reply pkg/webui/console/views/overview/index.js]: @bafonins, I've talked to @laurensslats, support links should be the following:
ttn: thethingsnetwork.org/support
ttes: thethingsindustries.com/support

File: pkg/webui/components/footer/index.js [hunk 7]
```diff
@@ -50,9 +50,14 @@ const Footer = function({ className, links, supportLink, isOnline }) {
         ))}
         <OfflineStatus isOnline={isOnline} showOfflineOnly showWarnings />
         <span className={style.version}>v{process.env.VERSION}</span>
+        {documentationLink && (
+          <Link.Anchor className={style.documentation} secondary href={documentationLink}>
```
Comment (bafonins) [inline pkg/webui/components/footer/index.js]:
```suggestion
          <Link.Anchor className={style.documentation} secondary href={documentationLink} external>
```

File: pkg/webui/console/views/overview/index.js [hunk 8]
```diff
@@ -247,6 +249,23 @@ export default class Overview extends React.Component {
                   component="h2"
                 />
               )}
+              <Message
+                className={style.getStarted}
+                content={supportLink ? m.needHelp : m.needHelpWithoutSupport}
```
Comment (bafonins) [inline pkg/webui/console/views/overview/index.js]: What if there is no link to docs?
We do check for it in [footer](https://github.com/TheThingsNetwork/lorawan-stack/blob/issue/2770-add-documentation-link/pkg/webui/components/footer/index.js#L53-L57). IMO, we should check for it here as well.

**Review by bafonins — CHANGES_REQUESTED** [review]

**Review by kschiffer — APPROVED** [review]

**Review by bafonins — APPROVED** [review]

**Pull request merged**